src/universal/modules/meeting/containers/MeetingContainer/MeetingContainer.js

```diff
@@ -65,6 +65,8 @@ export default class MeetingContainer extends Component {
       teamMembers !== this.props.memberSub.data.teamMembers ||
       team.activeFacilitator !== oldTeam.activeFacilitator ||
       user.id !== this.props.user.id) {
+      // We've changed teams, reset our container's state:
+      this.resetContainer();
```

Comment:
this works?
since team members come in as a stream, this gets called any time a new team member joins a team, or someone is promoted to facilitator, or the unlikely occurrence that the userId changes.

I think we'll want to remove this.